guibot/region.py

```diff
@@ -1161,13 +1312,22 @@ def fill_at(self, anchor: "Match | Location | Target | str",
 
         return self
 
-    def select_at(self, anchor: "Match | Location | Target | str",
-		  image_or_index: str | int, dx: int, dy: int, dw: int = 0,
-		  dh: int = 0, ret_flag: bool = True, mark_clicks: int = 1,
-		  tries: int = 3) -> "Region":
+    def select_at(
+        self,
+        anchor: "Match | Location | Target | str",
+        image_or_index: str | int,
+        dx: int,
+        dy: int,
+        dw: int = 0,
+        dh: int = 0,
+        ret_flag: bool = True,
+        mark_clicks: int = 1,
+        tries: int = 3,
+    ) -> "Region":
         """
-        Select an option at a dropdown list using either an integer index
-        or an option image if the order cannot be easily inferred.
+        Select an option at a dropdown list using either an integer index.
+
+        Or select an option image if the order cannot be easily inferred.
```

Comment:
Select an option at a dropdown list using an index or an image.

The caller can use either integer index or an option image if the order cannot be easily inferred.

Reply:
Done